spec/controllers/identities_controller_spec.rb

```diff
@@ -102,5 +102,31 @@ def reset_password(password, confirmation)
 
     end
 
+    context 'send_add' do
+
+      it 'redirects to the home page with a message if the user is not found' do
+        post :send_add
+        expect(response).to redirect_to root_path
+        expect(flash.alert).to eq I18n.t(:'controllers.identities.lost_user')
+      end
+
+      xit 'sends a message to add a password to the account if the user is found' do
+      end
```

Comment:
These `xit` success cases are already in `spec/features/reset_password_spec.rb`.  Want to remove them from here?